### DISCUSSION TITLE
Allow to set the dates manually

### DISCUSSION THREAD

**Discussion by underdpt** [malzariey/filament-daterangepicker-filter#143] [Ideas]
Hello,

I see that the input is actually set as readonly. Why is that? Would you consider allowing it to be editable?

**Comment by malzariey** [comment]
Thank you for posting in the discussion, 

As the input is constructed using a specific format [Date Seperator Date]. Managing the format is excessively complex.

As of now there is no plan to support this feature, but I do welcome any PRs.
